srcpkgs/prometheus/template

```diff
@@ -1,14 +1,15 @@
 # Template file for 'prometheus'
 pkgname=prometheus
-version=2.43.0
-revision=3
+version=2.48.1
+revision=1
 build_style=go
 go_import_path="github.com/prometheus/prometheus"
 go_package="github.com/prometheus/prometheus/cmd/prometheus github.com/prometheus/prometheus/cmd/promtool"
-go_ldflags="-X ${go_import_path}/version.Version=${version}
- -X ${go_import_path}/version.Revision=${version}
- -X ${go_import_path}/version.Branch=${version}
- -X ${go_import_path}/version.BuildUser=VoidLinux"
+go_ldflags="-X github.com/prometheus/common/version.Version=${version}
+ -X github.com/prometheus/common/version.Revision=${version}
+ -X github.com/prometheus/common/version.Branch=${version}
+ -X github.com/prometheus/common/version.BuildDate=$(date -u "+%Y%m%d-%H:%M:%S")
```

Comment:
date --date "@$SOURCE_DATE_EPOCH" "+%Y%m%d-%H:%M:%S"

Reply:
Ahh nice. Didn't realize we set that.